src/main/java/frc/utils/digitalinput/chooser/ChooserDigitalInput.java

```diff
@@ -0,0 +1,48 @@
+package frc.utils.digitalinput.chooser;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.utils.digitalinput.DigitalInputInputsAutoLogged;
+import frc.utils.digitalinput.IDigitalInput;
+
+public class ChooserDigitalInput implements IDigitalInput {
+
+	private final SendableChooser<Boolean> isTrueSendableChooser;
```

Comment:
rename to something like booleanSendableChoose or just sendableChooser

Reply:
or you can call it inputValueChooser/digitalInputValueChooser